### DISCUSSION TITLE
大量数据写入出错

### DISCUSSION THREAD

**Discussion by a3225167** [apache/doris#13825] [A - General / Q&A]
Spring boot + mybatis，批量写入。 会出现以下错误。版本是1.1.3。
需要怎么处理？参数调优吗？

<img width="234" alt="image" src="https://user-images.githubusercontent.com/32891043/198976441-d149d439-e0a4-414c-b7ca-d0afe7df15c8.png">

java.sql.SQLException: errCode = 2, detailMessage = wait close failed. tablet writer write failed, tablet_id=11781, txn_id=286013, err=-235, host: 10.0.0.123, host: 10.0.0.123
<img width="1653" alt="image" src="https://user-images.githubusercontent.com/32891043/198976223-75aa4475-4783-4163-bddd-404863caa0ff.png">

**Comment by LemonLiTree** [comment]
这是由于写入过快导致tablet的数据版本超过了最大限制，可以通过调大参数max_tablet_version_num，默认500。
另外也需要提升 compaction 的速度，compaction优化可以参考：https://mp.weixin.qq.com/s/cZmXEsNPeRMLHp379kc2aA

**Reply by a3225167** [reply]
我这看见这三个参数啊，没有看见max_compaction_threads
max_base_compaction_threads
max_cumu_compaction_threads
min_compaction_threads

**Reply by dataalive** [reply]
现在写入的时候是多少条一个批次？大约是个什么频率？
